kotlin-ring-ui/src/main/kotlin/ringui/Loader.ext.kt

```diff
@@ -0,0 +1,11 @@
+package ringui
+
+data class RGBImpl(override val r: Short = 0, override val g: Short = 0, override val b: Short = 0) : RGB
+
+fun RGB(r: Short, g: Short, b: Short): RGB = RGBImpl(r, g, b)
+
+var LoaderProps.dataTest: String
+    set(value) {
+        asDynamic()["data-test"] = value
+    }
+    get() = asDynamic()["data-test"] as? String ?: ""
```

Comment:
`get` first?
Is fallback required?

Reply:
Depending on what means fallback :)

Reply:
`""` - redundant fallback, cast also redundant
Valid type - `String?`